.gitignore

```diff
@@ -154,6 +154,9 @@ dmypy.json
 # Results folders
 **/results/
 
+# LM folders
+recipes/*/*/*/lm/
```

Comment:
Can we add a comment saying that this is for k2? I don't see any reason why other non-k2 recipe would have such a folder .